modules/apps/commerce/commerce-service/service.xml

```diff
@@ -378,6 +378,7 @@
 		<column name="subscriptionType" type="String" />
 		<column name="subscriptionTypeSettings" type="String" />
 		<column name="unitOfMeasureKey" type="String" />
+		<column db-name="unitOfMeasureQuantity" name="unitOfMeasureIncrementalOrderQuantity" type="BigDecimal" />
```

Comment:
can we call it `UOMIncrementalOrderQuantity`?

Reply:
Done